lib/features/children/generosity_challenge/pages/generosity_challenge_overview.dart

```diff
@@ -202,14 +205,23 @@ class _GenerosityChallengeOverviewState
                 GivtElevatedButton(
                   onTap: () {
                     context.read<FlowsCubit>().startInGenerosityCoinFlow();
-                    context.pushNamed(FamilyPages.scanNFC.name);
+                    context.pushNamed(
+                      FamilyPages.scanNFC.name,
+                      extra: {
+                        'isGenerosityChallenge': true,
+                      },
+                    );
                     AnalyticsHelper.logEvent(
                       eventName: AmplitudeEvents.giveWithCoinInChallengeClicked,
                     );
                   },
                   leadingImage: coin(width: 32, height: 32),
                   text: 'Give with a coin',
                 ),
+                if (Platform.isAndroid)
```

Comment:
This shouldn't be based on platform. I think it has to do with SafeArea, which automatically adds padding to phones that need it :)

Reply:
solution could be min bottom padding in safearea object or padding before safearea

Reply:
Agreed, fixed.